products/drush.md

```diff
@@ -26,13 +26,20 @@ auto:
         releaseCycle:
           column: "Drush Version"
           regex: '^Drush (?P<value>\d+).*$'
-          regex_exclude: '^Drush 13.*$' # not yet released
         supportedPhpVersions: "PHP Version" # release_table does not support the retrieval of Drupal versions
         eol: "End Of Life"
 
 # Think about updating the regex_exclude above when a new major version is added.
 # Supported PHP / Drupal versions can be found on https://www.drush.org/latest/install/.
 releases:
+-   releaseCycle: "13"
+    releaseDate: 2024-07-18
+    eol: false
+    supportedPhpVersions: "8.3+"
+    supportedDrupalVersions: "11"
```

Comment:
```suggestion
    supportedDrupalVersions: "10.2 - 11"
```

Reply:
Not sure we should mention 10.2 @chenrui333 : on https://www.drush.org/13.x/install/ you can see that drush 13 is compatible 10.2, but this is no longer supported nonetheless.

Do you think we should change this to only mention 11 ?